dpnp/dpnp_algo/dpnp_algo.pyx

```diff
@@ -337,26 +337,54 @@ cdef dparray call_fptr_1in_1out(DPNPFuncName fptr_name, dparray x1, dparray_shap
     return result
 
 
-cdef dparray call_fptr_2in_1out(DPNPFuncName fptr_name, dparray x1, dparray x2, dparray_shape_type result_shape, new_version=False):
+cdef dparray call_fptr_2in_1out(DPNPFuncName fptr_name, object x1_obj, object x2_obj,
+                                object dtype=None, dparray out=None, object where=True,
+                                bint new_version=False):
```

Comment:
better to add a comment here in case this param will be removed in the future.

Reply:
https://github.com/IntelPython/dpnp/blob/bd4206856b12c2279131c295ff85e68095c4be66/dpnp/dpnp_algo/dpnp_algo.pyx#L381